.codecov.yml

```diff
@@ -32,3 +32,4 @@ ignore:
   - "**/conf/**/*.php"
   - "**/*.css"
   - "**/*.scss"
+  - /library/deprecated/**.*.php
```

Comment:
Looking at the docs here https://docs.codecov.io/docs/ignoring-paths

This should not have a proceeding path.

And shouldn't it just be `library/deprecated`?